webpack/assets/javascripts/react_app/components/UserJwtToken/components/UserJwtTokenForm.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { Grid } from 'patternfly-react';
+import PropTypes from 'prop-types';
+
+import FormField from '../../common/forms/FormField';
+import Select from '../../common/forms/Select';
+import { translate as __ } from '../../../common/I18n';
+
+const UserJwtTokenForm = ({
+  expValue,
+  setExpValue,
+  expUnit,
+  setExpUnit,
+  handleSubmit,
+}) => {
+  const units = [
+    { label: __('Never'), value: 'never' },
+    { label: __('Hour'), value: 'hours' },
+    { label: __('Day'), value: 'days' },
+    { label: __('Week'), value: 'weeks' },
+    { label: __('Month'), value: 'months' },
+  ];
+
+  return (
+    <div>
+      <Grid.Row>
+        <Grid.Col md={8}>
+          <Select
+            label={__('Expire in')}
+            className="jwt-expiration-value"
+            options={units}
+            value={expUnit}
+            onChange={e => setExpUnit(e.target.value)}
+          />
```

Comment:
Can you use our `NumericInput` ?
cc @yifatmakias are there any expected changes in that component?

Reply:
@laviro Nop, Some changes were already merged and there are no more expected changes.